docs-src/index.html

```diff
@@ -327,7 +326,7 @@ <h2>Online <b class="underline">is optional</b></h2>
                 Next.js
             </div>
             <div
-                class="neumorphism-circle-m circle centered beating-second"
+                class="neumorphism-circle-m circle centeredd"
```

Comment:
There's a small typo here.

```suggestion
                class="neumorphism-circle-m circle centered"
```